ui/analyse/css/build/analyse.free.scss

```diff
@@ -1,4 +1,5 @@
 @import '../analyse.abstract';
 @import '../../../common/css/component/mselect';
+@import '../../../common/css/form/form3';
```

Comment:
the analysis board doesn't need form components, so I think the fieldset styling should be extracted to a new scss file, which form3 imports.

Reply:
Extracted to a reusable scss file: `ui/common/css/component/_toggle-box.scss`
Updated.